test/common/testlib.py

```diff
@@ -1219,6 +1218,43 @@ def sed_file(self, expr, path, apply_change_action=None):
                 self.addCleanup(m.execute, apply_change_action)
             self.addCleanup(m.execute, "mv {0}.cockpittest {0}".format(path))
 
+    def restore_dir(self, path, post_restore_action=None):
```

Comment:
Let's add a `reboot=` argument; if true, it uses classical `cp -a` instead of bind mounts.

Reply:
Implemented in https://github.com/cockpit-project/cockpit/pull/13758/commits/f71ef5bd8a4bf5ecc389df3a838defe4f638518b